Makefile

```diff
@@ -131,3 +131,7 @@ export_coverage:
 .PHONY: measure
 measure:
 	tools/measure
+	
+.PHONY: xcode
+xcode:
+	open Package.swift
```

Comment:
Nice! Can you include this change in a new PR? It'd be nice to get this change merged while we decide the path forward on this PR.

Reply:
Sure, no problem!